### PR TITLE
Substrate companion #11387

### PR DESCRIPTION
**Pull request by bkchr**
https://github.com/paritytech/substrate/pull/11387

### CODE REVIEW

File: runtime/common/src/crowdloan/migration.rs [hunk 1]
```diff
@@ -15,26 +15,27 @@
...
 
 /// Migrations for using fund index to create fund accounts instead of para ID.
 pub mod crowdloan_index_migration {
 	use super::*;
 
+	#[storage_alias]
+	type NextTrieIndex<T: Config> = StorageValue<Pallet<T>, FundIndex>;
+
+	#[storage_alias]
+	type Leases<T: Config> = StorageMap<Pallet<T>, Twox64Concat, ParaId, Vec<Option<(<T as frame_system::Config>::AccountId, BalanceOf<T>)>>>;
```
Comment (KiChjang) [inline runtime/common/src/crowdloan/migration.rs:28]: Is the prefix for this correct? Previously it had `Slots`, but this one now has `Pallet<T>`, which I assume is the one for `Crowdloan`?

Comment (bkchr) [reply runtime/common/src/crowdloan/migration.rs:28]: Ohh! Could find!

Comment (bkchr) [inline runtime/common/src/crowdloan/migration.rs:28]:
```suggestion
	type Leases<T: Config> = StorageMap<Slots, Twox64Concat, ParaId, Vec<Option<(<T as frame_system::Config>::AccountId, BalanceOf<T>)>>>;
```

**Review by shawntabrizi — APPROVED** [review]

**Review by kianenigma — APPROVED** [review]

**Review by KiChjang — APPROVED** [review]

**Comment by shawntabrizi** [comment]
bot merge

**Pull request merged**